app.py

```diff
@@ -41,6 +41,8 @@ def sign_up():
 		# would be great to have logs here
 		print("Form could not load, server error!")
 		pass
+	else:
```

Comment:
@michaelyaacoub this isn't correct - where it says `finally` it needs to say `else`, and where it says `pass` it needs to say `return jsonify(success=False)`.

Reply:
That makes sense, I will get it done today!